text_formatter.go

```diff
@@ -56,13 +56,23 @@ type TextFormatter struct {
 	// Whether the logger's out is to a terminal
 	isTerminal bool
 
+	// Format the message a particular way. The default is ' %-44s '.
+	MessageFormat string
+
+	// Disable logging empty messages. If set to true empty messages will not be
+	// printed in the log statement
+	DisableEmptyMessages bool
+
 	sync.Once
 }
 
 func (f *TextFormatter) init(entry *Entry) {
 	if entry.Logger != nil {
 		f.isTerminal = IsTerminal(entry.Logger.Out)
 	}
+	if f.MessageFormat == "" {
```

Comment:
The thing here is that this doesn't really change the message format. It only changes the length of the space between the timestamp and the actual message. The naming is really not good.